examples/pipeline/kg_builder_example.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright (c) "Neo4j"
```

Comment:
I think at some point we talked about not having this header in the examples? (I know some examples have it, but I'm asking because I was about to clean that when cleaning the folder)

Reply:
Good point. Perhaps we include it? I'm not sure when it makes sense not to include it